Changelog.md

```diff
@@ -1,4 +1,5 @@
 # Next
+- Added tests for Single+Response and refactored Shared Test Helpers.
```

Comment:
I don't know if adding tests needs a CHANGELOG entry but we usually list all API breaking changes first.